datalad/distribution/create_sibling.py

```diff
@@ -362,6 +363,17 @@ def _create_dataset_sibling(
                   " and run with --existing=reconfigure",
                   shell.get_git_version())
 
+    branch = ds_repo.get_active_branch()
+    if branch is not None:
+        branch = ds_repo.get_corresponding_branch(branch) or branch
```

Comment:
I guess just a note to myself: in 0.12.2-457-g8473249e8 behavior of get_corresponding_branch has changed, so now it is `None` if originally was not a managed branch. Code should still work if applied to maint where it would return the branch if not a managed one